test/cutting/test_find_cuts.py

```diff
@@ -0,0 +1,80 @@
+# This code is a Qiskit project.
+
+# (C) Copyright IBM 2024.
+
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for find_cuts module."""
+
+import unittest
+
+import pytest
+import os
+import numpy as np
+from qiskit import QuantumCircuit
+
+from circuit_knitting.cutting import (
+    find_cuts,
+    OptimizationParameters,
+    DeviceConstraints,
+)
+
+
+class TestCuttingDecomposition(unittest.TestCase):
+    def test_find_cuts(self):
+        with self.subTest("simple circuit"):
+            path_to_circuit = os.path.join(
+                os.path.dirname(__file__),
+                "..",
+                "qasm_circuits/circuit_find_cuts_test.qasm",
```

Comment:
Your tests pass so it's fine, but fyi this is technically preferable so that it will work on a hypothesized platform that uses a path separator other than `/`.  (Historically, this would have included Windows, which uses `\`, but CI passes on Windows, so you're good in that specific case and all that we actually care about.)

```suggestion
                "qasm_circuits",
                "circuit_find_cuts_test.qasm",
```

Reply:
Oh yeah that makes more sense. I don't even know why I left it like that. Will change this.

Reply:
'Tis done: [68b9c1c](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/471/commits/68b9c1cea37e54bb9f92ae0e85f02a927a111ef9)